webapp/channels/src/components/post_view/embedded_bindings/embedded_bindings.tsx

```diff
@@ -29,31 +29,27 @@ type Props = {
 
 }
 
-export default class EmbeddedBindings extends React.PureComponent<Props> {
-    static defaultProps = {
-        imagesMetadata: {},
-    };
-
-    render() {
-        const content = [] as JSX.Element[];
-        this.props.embeds.forEach((embed, i) => {
-            content.push(
-                <EmbeddedBinding
-                    embed={embed}
-                    post={this.props.post}
-                    key={'att_' + i}
-                    options={this.props.options}
-                />,
-            );
-        });
-
-        return (
-            <div
-                id={`messageAttachmentList_${this.props.post.id}`}
-                className='attachment__list'
-            >
-                {content}
-            </div>
+const EmbeddedBindings = ({embeds, post, options}: Props) => {
+    const content = [] as JSX.Element[];
+    embeds.forEach((embed, i) => {
```

Comment:
Out of the scope of the task, but I would appreciate if you do it.
This can use a map instead of pushing into the array. And we may consider move that map to the return. As you feel it is more readable. For example:

```
const content = embeds.map(...);
return (<div>{content}</div>);
```
or...
```
return (
    <div>
        {embeds.map(...)}
    </div>
)
```